front/src/main.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import CssBaseline from '@mui/material/CssBaseline'
-import './index.css'
+// import './index.css'
```

Comment:
./index.cssのコメントアウトを消してほしいです！
```suggestion
import './index.css'
```